core/storage/trie/polkadot_trie_db/polkadot_trie_batch.hpp

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_CORE_STORAGE_TRIE_POLKADOT_TRIE_DB_POLKADOT_TRIE_BATCH_HPP_
+#define KAGOME_CORE_STORAGE_TRIE_POLKADOT_TRIE_DB_POLKADOT_TRIE_BATCH_HPP_
+
+#include "common/buffer.hpp"
+#include "storage/face/write_batch.hpp"
+#include "storage/trie/polkadot_trie_db/polkadot_trie_db.hpp"
+
+namespace kagome::storage::trie {
+  class PolkadotTrieBatch
+      : public face::WriteBatch<common::Buffer, common::Buffer> {
+    enum class Action { PUT, REMOVE };
+    struct Command {
+      Action action;
+      common::Buffer key;
+      common::Buffer value;  // empty for remove
```

Comment:
You can just
```suggestion
      common::Buffer value{};  // empty for remove
```
To avoid making a remove command as `{Action::REMOVE, your_key, Buffer{}}` - the last buffer will not have to be created explicitly in this case